dashboard/app/models/stage.rb

```diff
@@ -80,6 +80,7 @@ def summarize
           name: localized_name,
           title: localized_title,
           flex_category: localized_category,
+          lockable: !!lockable,
```

Comment:
Nowhere yet. Later it will be used to show teachers some UI around allowing them to lock/unlock the stage.